book/C-git-commands/1-git-commands.asc

```diff
@@ -364,143 +364,146 @@ Pinakita natin paano gamitin ito para gumawa ng magandang changelog sa <<_distri
 
 ==== git describe
 
-The `git describe` command is used to take anything that resolves to a commit and produces a string that is somewhat human-readable and will not change.
-It's a way to get a description of a commit that is as unambiguous as a commit SHA-1 but more understandable.
-
-We use `git describe` in <<_distributed_git#_build_number>> and <<_distributed_git#_preparing_release>> to get a string to name our release file after.
+Ang `git describe` na utos ay ginamit para kunin ang kahit na anong paglutas sa isang commit at gumagawa ng isang string na nababasa ng tao at hindi ito magbabago.
+Ito ay isang paraan para kunin ang paglalarawan ng isang commit na hindi malabo kagaya ng commit na SHA-1 ngunit mas naiintindihan.
 
+Ginamit natin ang `git describe` sa <<_distributed_git#_build_number>> at 
+<<_distributed_git#_preparing_release>> para kumuha ng string para pangalanan ang ating pinakawalang file.
 
 === Debugging
 
-Git has a couple of commands that are used to help debug an issue in your code.
-This ranges from figuring out where something was introduced to figuring out who introduced it.
+Ang Git ay may kakarampot na mga utos na ginamit para makatulong sa pag-debug ng problema sa code mo.
```

Comment:
I think `Ang Git ay may kakarampot na mga utos na ginamit para makatulong sa pag-debug ng problema sa code mo.` should be changed to `Ang Git ay may iilang mga utos na ginamit para makatulong sa pag-debug sa isyu sa iyong code.`